packages/shared/src/contexts/NotificationsContext.tsx

```diff
@@ -2,16 +2,20 @@ import React, {
   ReactElement,
   ReactNode,
   useContext,
+  useEffect,
   useMemo,
   useState,
 } from 'react';
+import OneSignalReact from 'react-onesignal';
```

Comment:
We need to change the import of this library to dynamic import to reduce the bundle size.
Let me know if you want more information